api/rds.proto

```diff
@@ -48,7 +56,7 @@ message WeightedCluster {
   message ClusterWeight {
     // Name of the upstream cluster. The cluster must exist in the cluster
     // manager configuration.
-    google.protobuf.StringValue name = 1;
+    string name = 1;
```

Comment:
Yeah, I'm not aware of any either. I don't mind making this restriction, so after @louiscryan's patch goes in I can update the other google.protobuf.StringValues.